Encounter Me/Pages/Leaderboard.razor

```diff
@@ -1,43 +1,100 @@
 ﻿@page "/Leaderboard"
 
 
-<h1 class="page title">Leaderboard</h1>
+@if (Users == null)
+{
+    <p><em>Loading...</em></p>
+}
+else
+{
+    <div class="container" style="margin-top:10%">
+        <div class="row">
+            <div style="margin:auto">
+                <a href="@($"Leaderboard")" style="">Users</a>
+                |
+                <a href="@($"Leaderboard/ByFactions")">Faction View</a>
+                |
+                <a href="@($"Leaderboard/Factions")">Factions</a>
+            </div> 
+            <div style="margin-right:5%">
+                <select style="width:50px;float:right" @onclick="@(() => doUsers5(pageNumber))" @bind=@pageNumber>
+                    @for (int p = 1; p <= (Users.Count() / 5); p++)
+                    {
+                        <option value="@p">@p</option>
+                    }
+                    @if (Users.Count() % 5 != 0)
+                    {
+                        <option value=@((Users.Count() / 5)+1)>@((Users.Count() / 5)+1)</option>
+                    }
```

Comment:
What is this drop down for?

Reply:
It generates, number of pages to check out all leaaderboard.